guacamole-common-js/src/main/webapp/modules/Client.js

```diff
@@ -1416,8 +1450,24 @@ Guacamole.Client = function(tunnel) {
         },
         
         "msg" : function(parameters) {
-            
-            if (guac_client.onmsg) guac_client.onmsg(parseInt(parameters[0]), parameters.slice(1));
+
+            var msgid = parseInt(parameters[0]);
+            if (guac_client.onmsg)
+                guac_client.onmsg(msgid, parameters.slice(1));
+
+            switch (msgid) {
+
+                case Guacamole.Client.Message.USER_JOINED:
+                    if (guac_client.onjoin)
+                        guac_client.onjoin(parameters[1] /* User ID */, parameters[2] /* Name */);
```

Comment:
These comments are helpful to know what these parameters are, but why not just put the values in appropriately named variables? That seems more straightforward.

Reply:
Sure.

Reply:
Switched over to variables with a new commit.